pkg/adapter/adapter.go

```diff
@@ -295,21 +293,21 @@ func (a *Adapter) postMessage(msg wabbit.Delivery) error {
 		return err
 	}
 
+	// TODO: See if the message is already a Cloud Event and if so, do not wrap, just use as is.
```

Comment:
You can grab the logic for it from kafka source, we have a pretty much similar one https://github.com/knative-sandbox/eventing-kafka/blob/master/pkg/source/adapter/message.go#L39